devtools/maven/src/main/java/io/quarkus/maven/CreateProjectMojo.java

```diff
@@ -83,12 +78,21 @@
     @Parameter(property = "projectVersion")
     private String projectVersion;
 
+    /**
+     * Group ID of the target platform BOM
+     */
     @Parameter(property = "platformGroupId", defaultValue = CreateUtils.DEFAULT_PLATFORM_BOM_GROUP_ID)
     private String bomGroupId;
 
-    @Parameter(property = "platformArtifactId", defaultValue = CreateUtils.DEFAULT_PLATFORM_BOM_ARTIFACT_ID)
+    /**
+     * Artifact ID of the target platform BOM
+     */
+    @Parameter(property = "platformArtifactId", required = false)
     private String bomArtifactId;
 
+    /**
+     * Version of the target platform BOM
+     */
```

Comment:
This is treated as a "Version or version range of the target platform" which is then used to resolve bom and/or json, right?

Reply:
hmm - where are the limits for the version range search actually set ?

Reply:
`bomVersion` is the actual version. If it's not set then the latest one will be resolved.
The range is set here 
https://github.com/quarkusio/quarkus/pull/5268/files#diff-20e78dc83510b6617d2789e2e291bb1aR235
and here
https://github.com/quarkusio/quarkus/pull/5268/files#diff-20e78dc83510b6617d2789e2e291bb1aR429
The range can be customized using this property https://github.com/quarkusio/quarkus/blob/master/independent-projects/tools/platform-descriptor-resolver-json/src/main/java/io/quarkus/platform/descriptor/resolver/json/QuarkusJsonPlatformDescriptorResolver.java#L51
This can definitely be improved.

Reply:
yeah so this feels a tad confusing. this why I changed the names of what the user requests at create time to be about what platform he is looking for not wether it’s a bom or json. That it maps directly to a bom is a implementation detail imo ;) 

is there a reason to not make the `platformVersion` parameter version range and be applicable no matter if artifact/groupid . are specified ? (since exact version is possible there too) ?

Reply:
Yes, it could be convenient to make it a range as well.
About requiring the artifactId if the version is provided, I didn't like making assumptions about what the default artifactId is in the mojo. But that's arguable and I won't mind not requiring it.

Reply:
Ok, it's done. You can provide a range through `platformVersion` (quoted to get it accepted as a cmd arg) and `platformArtifactId` isn't required.